curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fefebad99209211ec30537.md

```diff
@@ -0,0 +1,404 @@
+---
+id: 64fefebad99209211ec30537
+title: Step 49
+challengeType: 0
+dashedName: step-49
+---
+
+# --description--
+
+If you check the `Application` tab of your browser console, you'll see a series of `[object Object]`. That's because everything you save in `localStorage` has to be a string.
```

Comment:
```suggestion
If you check the `Application` tab of your browser console, you'll notice a series of `[object Object]`. This is because everything you save in `localStorage` needs to be in string format.
```